Formula/p/pypy3.10.rb

```diff
@@ -28,11 +28,11 @@ class Pypy310 < Formula
   depends_on "sqlite"
   depends_on "tcl-tk"
   depends_on "xz"
+  depends_on "ncurses" # for ncursesw
```

Comment:
This shouldn't be necessary as build will check for multiple different names. May be hitting `-Wincompatible-function-pointer-types` failure instead like what I saw in #164607